state/backend.go

```diff
@@ -28,10 +28,19 @@ type Writer interface {
 // The appropriate backend
 func InitBackend() Backender {
 	cfg = config.GetConfig()
-	switch cfg.StateBackend.Type {
+	s := NewEmptyState()
+	switch {
+	case config.Contains(config.StateBackendTypes[1:], cfg.StateBackend.Type):
```

Comment:
Could we reference the `RemoteBackendTypes` from here, instead of using indexes. It could get messy and difficult to debug